src/componets/ui/button/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
+import React, { useState } from 'react';
+
+import { ButtonGroup, ButtonToggle } from './button.styled';
+
+const types = ['All', 'Movies', 'TV-Shows'];
+
+const ToggleGroup = () => {
+  const [active, setActive] = useState(types[0]);
```

Comment:
It will work if you open the "All" page and goes to another page by the menu. What if you open the "Movies" page and make refresh the page? Please fix it. Could you check the browser URL and compare it with the menu?